### PR TITLE
Mark Set-Service tests with password as `Pending`

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Fix another install of non-complex password.

## PR Context

Can cause CI to fail.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by SteveL-MSFT — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 [hunk 1]
```diff
@@ -168,8 +168,7 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
             $startUsername = "user1"
```
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Get-Service can get the 'UserName' of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 303
303:             $service | Should -Not -BeNullOrEmpty
```

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 [hunk 2]
```diff
@@ -168,8 +168,7 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
...
             $endUsername = "user2"
             $servicename = "testsetcredential"
-            $Password = ([char[]]([char]33..[char]95) + ([char[]]([char]97..[char]126)) + 0..9 | Sort-Object {Get-Random})[0..12] -join ''
-            $testPass = (New-Object -TypeName Net.NetworkCredential("", $Password)).SecurePassword
+            $testPass = [Net.NetworkCredential]::new("", (New-ComplexPassword)).SecurePassword
             $creds = [pscredential]::new(".\$endUsername", $testPass)
             net user $startUsername $creds.GetNetworkCredential().Password /add > $null
```
Comment (SteveL-MSFT) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: Let's not redirect this to $null so we can see the output and confirm if this is still a password complexity issue

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 (hunk 1 above)
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG.md.https://dependabot.com should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```

**Comment by adityapatwardhan** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-static-analysis`

**Review by SteveL-MSFT — APPROVED** [review]
Ok to unblock release

**Comment by adityapatwardhan** [comment]
Filed issue to track the failure: https://github.com/PowerShell/PowerShell/issues/10147

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 (hunk 1 above)
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: @adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG.md.https://dependabot.com should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG.md.https://dependabot.com should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```

**Comment by adityapatwardhan** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-static-analysis`

File: test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1 (hunk 1 above)
Comment (PoshChan) [inline test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Set-Service can change credentials of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 182
182:             $service | Should -Not -BeNullOrEmpty
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@PoshChan please retry linux

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-Linux`

**Pull request merged**